src/openforms/submissions/tests/test_disabled_forms.py

```diff
@@ -42,6 +43,11 @@ class InactiveFormTests(SubmissionsMixin, APITestCase):
     Tests for forms that are not active.
     """
 
+    def setUp(self):
+        # clear cache to avoid throttling
+        super().setUp()
+        cache.clear()
```

Comment:
is this still needed if you fix the setting overrides that broke isolation? If it is, please use the existing utility: https://github.com/open-formulieren/open-forms/blob/master/src/openforms/utils/tests/cache.py#L4

this should also definitely be added as a cleanup in your throttle tests: `self.addCleanup(clear_caches)` which is the _source_ of test isolation breaking (after the tests ran you need some cleanup/cache reset).

You can also isolate those tests/caches by specifying a different cache, like the in memory cache via `@override_settings(CACHES=...)`.